solidity/test/base/Constants.t.sol

```diff
@@ -69,13 +80,15 @@ contract ConstantsTest is Test {
 
     function testModulusPlusOneIsCorrect() public pure {
         assert(MODULUS_PLUS_ONE == MODULUS + 1);
+        assert(MODULUS_MINUS_ONE == MODULUS - 1);
```

Comment:
Please fix the function name now that we check that MODULUS_MINUS_ONE is correct as well